packages/application-generic/src/services/distributed-lock/distributed-lock.service.spec.ts

```diff
@@ -1,63 +1,71 @@
-import { expect } from 'chai';
-import * as sinon from 'sinon';
-import type { SinonSpy } from 'sinon';
-import Redis from 'ioredis';
-import * as Redlock from 'redlock';
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import Redlock from 'redlock';
 import { setTimeout } from 'timers/promises';
 
-import { InMemoryProviderClient, InMemoryProviderService } from '../in-memory-provider';
+import {
+  InMemoryProviderClient,
+  InMemoryProviderService,
+} from '../in-memory-provider';
 import { DistributedLockService } from './distributed-lock.service';
 
-const originalRedisCacheServiceHost = process.env.REDIS_CACHE_SERVICE_HOST;
-const originalRedisCacheServicePort = process.env.REDIS_CACHE_SERVICE_PORT;
+const originalRedisCacheServiceHost = (process.env.REDIS_CACHE_SERVICE_HOST =
+  process.env.REDIS_CACHE_SERVICE_HOST ?? 'localhost');
+const originalRedisCacheServicePort = (process.env.REDIS_CACHE_SERVICE_PORT =
+  process.env.REDIS_CACHE_SERVICE_PORT ?? '6379');
```

Comment:
The fallbacks shouldn't be needed.

Reply:
from what I remember they are needed, otherwise tests are not passing in the `application-generic` package

Reply:
checked that, the fallback is needed

Reply:
Some environment variables are not set in the CI then. Which file is being pulled?